docs/sources/operators-guide/configuring/configuring-shuffle-sharding/index.md

```diff
@@ -194,6 +195,12 @@ When you enable compactor shuffle sharding by setting `-compactor.compactor-tena
 
 You can override the compactor shard size on a per-tenant basis setting by `compactor_tenant_shard_size` in the overrides section of the runtime configuration.
 
+### Alertmanager shuffle sharding
+
+Alertmanager only distributes across replicas by tenant, the state and workload it is not divided any further. The replication factor setting `-alertmanager.sharding-ring.replication-factor` dictates how many replicas are used for a tenant.
```

Comment:
```suggestion
Alertmanager only performs distribution across replicas per tenant. The state and workload is not divided any further. The replication factor setting `-alertmanager.sharding-ring.replication-factor` determines how many replicas are used for a tenant.
```

Reply:
Please vet for technical accuracy because the first sentence seems like it initially contained a typo.

Reply:
The verb "to distribute" requires a direct object.

Reply:
You're right, there was an extra "it".